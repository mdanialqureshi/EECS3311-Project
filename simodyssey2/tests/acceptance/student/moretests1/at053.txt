-- Test (test mode) land, liftoff, move
test(1,1,1,1,30)
move(S)
move(S)
land
liftoff
move(E)