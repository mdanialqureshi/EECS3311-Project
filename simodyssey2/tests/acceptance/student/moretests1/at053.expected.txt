  state:0.0, ok
  Welcome! Try test(1,1,1,1,30)
->test(1,1,1,1,30)
  state:1.0, mode:test, ok
  Movement:none
  Sectors:
    [1,1]->[0,E],-,-,-
    [1,2]->[-8,*],-,-,-
    [1,3]->[1,P],-,-,-
    [1,4]->-,-,-,-
    [1,5]->[2,P],[3,P],-,-
    [2,1]->-,-,-,-
    [2,2]->-,-,-,-
    [2,3]->[-5,W],-,-,-
    [2,4]->-,-,-,-
    [2,5]->[4,P],-,-,-
    [3,1]->[5,P],[-6,Y],-,-
    [3,2]->-,-,-,-
    [3,3]->[-1,O],-,-,-
    [3,4]->[6,P],[-11,W],-,-
    [3,5]->-,-,-,-
    [4,1]->[-2,W],-,-,-
    [4,2]->[7,P],[-7,*],-,-
    [4,3]->[8,P],-,-,-
    [4,4]->[9,P],[-4,*],-,-
    [4,5]->-,-,-,-
    [5,1]->[10,P],[-3,W],-,-
    [5,2]->-,-,-,-
    [5,3]->[-10,*],-,-,-
    [5,4]->-,-,-,-
    [5,5]->[11,P],[-9,W],-,-
  Descriptions:
    [-11,W]->
    [-10,*]->Luminosity:5
    [-9,W]->
    [-8,*]->Luminosity:5
    [-7,*]->Luminosity:5
    [-6,Y]->Luminosity:2
    [-5,W]->
    [-4,*]->Luminosity:5
    [-3,W]->
    [-2,W]->
    [-1,O]->
    [0,E]->fuel:3/3, life:3/3, landed?:F
    [1,P]->attached?:F, support_life?:F, visited?:F, turns_left:2
    [2,P]->attached?:F, support_life?:F, visited?:F, turns_left:2
    [3,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [4,P]->attached?:F, support_life?:F, visited?:F, turns_left:2
    [5,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [6,P]->attached?:F, support_life?:F, visited?:F, turns_left:2
    [7,P]->attached?:F, support_life?:F, visited?:F, turns_left:2
    [8,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [9,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [10,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [11,P]->attached?:F, support_life?:F, visited?:F, turns_left:2
  Deaths This Turn:none
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    E---   *---   P---   ----   PP--   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    ----   ----   W---   ----   P---   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    PY--   ----   O---   PW--   ----   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    W---   P*--   P---   P*--   ----   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    PW--   ----   *---   ----   PW--   
->move(S)
  state:2.0, mode:test, ok
  Movement:
    [0,E]:[1,1,1]->[2,1,1]
    [3,P]:[1,5,2]->[5,4,1]
    [8,P]:[4,3,1]->[3,2,1]
  Sectors:
    [1,1]->-,-,-,-
    [1,2]->[-8,*],-,-,-
    [1,3]->[1,P],-,-,-
    [1,4]->-,-,-,-
    [1,5]->[2,P],-,-,-
    [2,1]->[0,E],-,-,-
    [2,2]->-,-,-,-
    [2,3]->[-5,W],-,-,-
    [2,4]->-,-,-,-
    [2,5]->[4,P],-,-,-
    [3,1]->[5,P],[-6,Y],-,-
    [3,2]->[8,P],-,-,-
    [3,3]->[-1,O],-,-,-
    [3,4]->[6,P],[-11,W],-,-
    [3,5]->-,-,-,-
    [4,1]->[-2,W],-,-,-
    [4,2]->[7,P],[-7,*],-,-
    [4,3]->-,-,-,-
    [4,4]->[9,P],[-4,*],-,-
    [4,5]->-,-,-,-
    [5,1]->[10,P],[-3,W],-,-
    [5,2]->-,-,-,-
    [5,3]->[-10,*],-,-,-
    [5,4]->[3,P],-,-,-
    [5,5]->[11,P],[-9,W],-,-
  Descriptions:
    [-11,W]->
    [-10,*]->Luminosity:5
    [-9,W]->
    [-8,*]->Luminosity:5
    [-7,*]->Luminosity:5
    [-6,Y]->Luminosity:2
    [-5,W]->
    [-4,*]->Luminosity:5
    [-3,W]->
    [-2,W]->
    [-1,O]->
    [0,E]->fuel:2/3, life:3/3, landed?:F
    [1,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [2,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [3,P]->attached?:F, support_life?:F, visited?:F, turns_left:2
    [4,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [5,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [6,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [7,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [8,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [9,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [10,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [11,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
  Deaths This Turn:none
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    ----   *---   P---   ----   P---   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    E---   ----   W---   ----   P---   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    PY--   P---   O---   PW--   ----   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    W---   P*--   ----   P*--   ----   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    PW--   ----   *---   P---   PW--   
->move(S)
  state:3.0, mode:test, ok
  Movement:
    [0,E]:[2,1,1]->[3,1,3]
    [8,P]:[3,2,1]->[4,1,2]
    [10,P]:[5,1,1]->[1,2,2]
  Sectors:
    [1,1]->-,-,-,-
    [1,2]->[-8,*],[10,P],-,-
    [1,3]->[1,P],-,-,-
    [1,4]->-,-,-,-
    [1,5]->[2,P],-,-,-
    [2,1]->-,-,-,-
    [2,2]->-,-,-,-
    [2,3]->[-5,W],-,-,-
    [2,4]->-,-,-,-
    [2,5]->[4,P],-,-,-
    [3,1]->[5,P],[-6,Y],[0,E],-
    [3,2]->-,-,-,-
    [3,3]->[-1,O],-,-,-
    [3,4]->[6,P],[-11,W],-,-
    [3,5]->-,-,-,-
    [4,1]->[-2,W],[8,P],-,-
    [4,2]->[7,P],[-7,*],-,-
    [4,3]->-,-,-,-
    [4,4]->[9,P],[-4,*],-,-
    [4,5]->-,-,-,-
    [5,1]->-,[-3,W],-,-
    [5,2]->-,-,-,-
    [5,3]->[-10,*],-,-,-
    [5,4]->[3,P],-,-,-
    [5,5]->[11,P],[-9,W],-,-
  Descriptions:
    [-11,W]->
    [-10,*]->Luminosity:5
    [-9,W]->
    [-8,*]->Luminosity:5
    [-7,*]->Luminosity:5
    [-6,Y]->Luminosity:2
    [-5,W]->
    [-4,*]->Luminosity:5
    [-3,W]->
    [-2,W]->
    [-1,O]->
    [0,E]->fuel:3/3, life:3/3, landed?:F
    [1,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [2,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [3,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [4,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [5,P]->attached?:T, support_life?:F, visited?:F, turns_left:N/A
    [6,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [7,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [8,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [9,P]->attached?:T, support_life?:F, visited?:F, turns_left:N/A
    [10,P]->attached?:T, support_life?:F, visited?:F, turns_left:N/A
    [11,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
  Deaths This Turn:none
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    ----   *P--   P---   ----   P---   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    ----   ----   W---   ----   P---   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    PYE-   ----   O---   PW--   ----   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    WP--   P*--   ----   P*--   ----   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    -W--   ----   *---   P---   PW--   
->land
  state:4.0, mode:test, ok
  Explorer found no life as we know it at Sector:3:1
  Movement:
    [1,P]:[1,3,1]->[2,4,1]
    [2,P]:[1,5,1]->[5,1,1]
    [4,P]:[2,5,1]->[2,1,1]
    [6,P]:[3,4,1]->[4,4,3]
    [11,P]:[5,5,1]->[5,4,2]
  Sectors:
    [1,1]->-,-,-,-
    [1,2]->[-8,*],[10,P],-,-
    [1,3]->-,-,-,-
    [1,4]->-,-,-,-
    [1,5]->-,-,-,-
    [2,1]->[4,P],-,-,-
    [2,2]->-,-,-,-
    [2,3]->[-5,W],-,-,-
    [2,4]->[1,P],-,-,-
    [2,5]->-,-,-,-
    [3,1]->[5,P],[-6,Y],[0,E],-
    [3,2]->-,-,-,-
    [3,3]->[-1,O],-,-,-
    [3,4]->-,[-11,W],-,-
    [3,5]->-,-,-,-
    [4,1]->[-2,W],[8,P],-,-
    [4,2]->[7,P],[-7,*],-,-
    [4,3]->-,-,-,-
    [4,4]->[9,P],[-4,*],[6,P],-
    [4,5]->-,-,-,-
    [5,1]->[2,P],[-3,W],-,-
    [5,2]->-,-,-,-
    [5,3]->[-10,*],-,-,-
    [5,4]->[3,P],[11,P],-,-
    [5,5]->-,[-9,W],-,-
  Descriptions:
    [-11,W]->
    [-10,*]->Luminosity:5
    [-9,W]->
    [-8,*]->Luminosity:5
    [-7,*]->Luminosity:5
    [-6,Y]->Luminosity:2
    [-5,W]->
    [-4,*]->Luminosity:5
    [-3,W]->
    [-2,W]->
    [-1,O]->
    [0,E]->fuel:3/3, life:3/3, landed?:T
    [1,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [2,P]->attached?:F, support_life?:F, visited?:F, turns_left:2
    [3,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [4,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [5,P]->attached?:T, support_life?:F, visited?:T, turns_left:N/A
    [6,P]->attached?:T, support_life?:F, visited?:F, turns_left:N/A
    [7,P]->attached?:T, support_life?:F, visited?:F, turns_left:N/A
    [8,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [9,P]->attached?:T, support_life?:F, visited?:F, turns_left:N/A
    [10,P]->attached?:T, support_life?:F, visited?:F, turns_left:N/A
    [11,P]->attached?:F, support_life?:F, visited?:F, turns_left:2
  Deaths This Turn:none
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    ----   *P--   ----   ----   ----   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    P---   ----   W---   P---   ----   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    PYE-   ----   O---   -W--   ----   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    WP--   P*--   ----   P*P-   ----   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    PW--   ----   *---   PP--   -W--   
->liftoff
  state:5.0, mode:test, ok
  Explorer has lifted off from planet at Sector:3:1
  Movement:
    [1,P]:[2,4,1]->[1,3,1]
    [3,P]:[5,4,1]->[1,3,2]
    [4,P]:[2,1,1]->[2,5,1]
    [8,P]:[4,1,2]->[5,5,1]
  Sectors:
    [1,1]->-,-,-,-
    [1,2]->[-8,*],[10,P],-,-
    [1,3]->[1,P],[3,P],-,-
    [1,4]->-,-,-,-
    [1,5]->-,-,-,-
    [2,1]->-,-,-,-
    [2,2]->-,-,-,-
    [2,3]->[-5,W],-,-,-
    [2,4]->-,-,-,-
    [2,5]->[4,P],-,-,-
    [3,1]->[5,P],[-6,Y],[0,E],-
    [3,2]->-,-,-,-
    [3,3]->[-1,O],-,-,-
    [3,4]->-,[-11,W],-,-
    [3,5]->-,-,-,-
    [4,1]->[-2,W],-,-,-
    [4,2]->[7,P],[-7,*],-,-
    [4,3]->-,-,-,-
    [4,4]->[9,P],[-4,*],[6,P],-
    [4,5]->-,-,-,-
    [5,1]->[2,P],[-3,W],-,-
    [5,2]->-,-,-,-
    [5,3]->[-10,*],-,-,-
    [5,4]->-,[11,P],-,-
    [5,5]->[8,P],[-9,W],-,-
  Descriptions:
    [-11,W]->
    [-10,*]->Luminosity:5
    [-9,W]->
    [-8,*]->Luminosity:5
    [-7,*]->Luminosity:5
    [-6,Y]->Luminosity:2
    [-5,W]->
    [-4,*]->Luminosity:5
    [-3,W]->
    [-2,W]->
    [-1,O]->
    [0,E]->fuel:3/3, life:3/3, landed?:F
    [1,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [2,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [3,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [4,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [5,P]->attached?:T, support_life?:F, visited?:T, turns_left:N/A
    [6,P]->attached?:T, support_life?:F, visited?:F, turns_left:N/A
    [7,P]->attached?:T, support_life?:F, visited?:F, turns_left:N/A
    [8,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [9,P]->attached?:T, support_life?:F, visited?:F, turns_left:N/A
    [10,P]->attached?:T, support_life?:F, visited?:F, turns_left:N/A
    [11,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
  Deaths This Turn:none
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    ----   *P--   PP--   ----   ----   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    ----   ----   W---   ----   P---   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    PYE-   ----   O---   -W--   ----   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    W---   P*--   ----   P*P-   ----   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    PW--   ----   *---   -P--   PW--   
->move(E)
  state:6.0, mode:test, ok
  Movement:
    [0,E]:[3,1,3]->[3,2,1]
    [1,P]:[1,3,1]->[5,3,2]
    [3,P]:[1,3,2]->[2,2,1]
    [4,P]:[2,5,1]->[3,5,1]
  Sectors:
    [1,1]->-,-,-,-
    [1,2]->[-8,*],[10,P],-,-
    [1,3]->-,-,-,-
    [1,4]->-,-,-,-
    [1,5]->-,-,-,-
    [2,1]->-,-,-,-
    [2,2]->[3,P],-,-,-
    [2,3]->[-5,W],-,-,-
    [2,4]->-,-,-,-
    [2,5]->-,-,-,-
    [3,1]->[5,P],[-6,Y],-,-
    [3,2]->[0,E],-,-,-
    [3,3]->[-1,O],-,-,-
    [3,4]->-,[-11,W],-,-
    [3,5]->[4,P],-,-,-
    [4,1]->[-2,W],-,-,-
    [4,2]->[7,P],[-7,*],-,-
    [4,3]->-,-,-,-
    [4,4]->[9,P],[-4,*],[6,P],-
    [4,5]->-,-,-,-
    [5,1]->[2,P],[-3,W],-,-
    [5,2]->-,-,-,-
    [5,3]->[-10,*],[1,P],-,-
    [5,4]->-,[11,P],-,-
    [5,5]->[8,P],[-9,W],-,-
  Descriptions:
    [-11,W]->
    [-10,*]->Luminosity:5
    [-9,W]->
    [-8,*]->Luminosity:5
    [-7,*]->Luminosity:5
    [-6,Y]->Luminosity:2
    [-5,W]->
    [-4,*]->Luminosity:5
    [-3,W]->
    [-2,W]->
    [-1,O]->
    [0,E]->fuel:2/3, life:3/3, landed?:F
    [1,P]->attached?:T, support_life?:F, visited?:F, turns_left:N/A
    [2,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [3,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [4,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [5,P]->attached?:T, support_life?:F, visited?:T, turns_left:N/A
    [6,P]->attached?:T, support_life?:F, visited?:F, turns_left:N/A
    [7,P]->attached?:T, support_life?:F, visited?:F, turns_left:N/A
    [8,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [9,P]->attached?:T, support_life?:F, visited?:F, turns_left:N/A
    [10,P]->attached?:T, support_life?:F, visited?:F, turns_left:N/A
    [11,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
  Deaths This Turn:none
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    ----   *P--   ----   ----   ----   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    ----   P---   W---   ----   ----   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    PY--   E---   O---   -W--   P---   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    W---   P*--   ----   P*P-   ----   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    PW--   ----   *P--   -P--   PW--   
